lib/pos_mempool.go

```diff
@@ -219,16 +214,14 @@ func (mp *PosMempool) Init(
 	mp.latestBlockHeight = latestBlockHeight
 	mp.dir = dir
 	mp.inMemoryOnly = inMemoryOnly
-	mp.maxMempoolPosSizeBytes = maxMempoolPosSizeBytes
 	mp.mempoolBackupIntervalMillis = mempoolBackupIntervalMillis
 	mp.augmentedBlockViewRefreshIntervalMillis = augmentedBlockViewRefreshIntervalMillis
 
-	// TODO: parameterize num blocks. Also, how to pass in blocks.
 	err = mp.feeEstimator.Init(
 		mp.txnRegister,
-		feeEstimatorNumMempoolBlocks,
+		globalParams.MempoolFeeEstimatorNumMempoolBlocks,
```

Comment:
Since these are in the global params, we can just parse these from `mp.globalParams` in the Init function for the fee estimator.

Reply:
This also reminds me that we'll want to call `mp.feeEstimator.UpdateGlobalParams` in `mp.UpdateGlobalParams`

Reply:
+1 to both of these. I'm addressing the fee estimator side of the changes in the next PR. I wanted to scope this PR to just the mempool side of the changes